src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -217,18 +223,22 @@ extension ExposureSubmissionCoordinator {
 				style: .cancel,
 				handler: { _ in
 					alert.dismiss(animated: true)
-
 				}
 			)
 		)
 		alert.preferredAction = acceptAction
 
-		navigationController?.present(alert, animated: true)
+		navigationController?.present(alert, animated: true, completion: {
+			#if DEBUG
+			if isUITesting {
+				alert.applyAccessibilityIdentifiers()
+			}
+			#endif
```

Comment:
This would not be needed if we can simply set the ids per `UIAlertAction` AND are able to tap that action (which contains a button) in the ui test. Haven't tested that yet but am happy to help @KaiTeuber